js/glsl/scatter-vertex.glsl

```diff
@@ -35,55 +34,252 @@ varying vec4 vertex_color;
 varying vec3 vertex_position;
 varying vec2 vertex_uv;
 
-#ifdef AS_LINE
-attribute vec3 position_previous;
+#ifdef IS_LINE
+    attribute vec3 position_previous;
 #else
-attribute float x;
-attribute float x_previous;
-attribute float y;
-attribute float y_previous;
-attribute float z;
-attribute float z_previous;
-attribute float aux;
-attribute float aux_previous;
-
-attribute vec3 v;
-attribute vec3 v_previous;
+    attribute float x_next;
+    attribute float x_previous;
+    attribute float y_next;
+    attribute float y_previous;
+    attribute float z_next;
+    attribute float z_previous;
+    // attribute float aux_next;
+    // attribute float aux_previous;
```

Comment:
I had to disable aux, because we are using too many attributes (16 is the max), that is because if we set USE_COLOR, `color` is also an attribute. I'll think about how to solve this.